src/com/phronemophobic/membrane/term.clj

```diff
@@ -317,47 +320,59 @@
     (color-scheme/load-scheme source)
     default-color-scheme))
 
+(defn font-exists? [font-family font-size]
+  (or (= "monospace" font-family)
+      (skia/font-exists? (ui/font font-family font-size))))
+
+(defn- load-terminal-font
+  "No checking is done, but font is assumed to be monospaced with a constant advancement width."
+  [font-family font-size]
+  (let [term-font (if (font-exists? font-family font-size)
+                    (ui/font font-family font-size)
+                    (ui/font "monospace" 12))
+        metrics (skia/skia-font-metrics term-font)
+        baseline-offset (- (:Ascent metrics))
+        descent-offset (+ baseline-offset (:Descent metrics))]
+    {:term-font term-font
```

Comment:
What do you think of just `assoc`ing `:cell-width`, `:cell-height`, and `descent-gap` onto `term-font`?

Reply:
I had considered this, but forgot that you only lose record-ness when you dissoc, not when you assoc.  Ex:

```
(def f (ui/font "monospace" 12))

(type f)
;; => membrane.ui.Font
(type (assoc f :fooya 12))
;; => membrane.ui.Font
(type (dissoc f :name))
;; => clojure.lang.PersistentArrayMap
```

So yeah, great idea. I think the code would benefit.
Should we namespace new keys? `:membrane.term/cell-width`, etc?

Reply:
The recordness isn't even important. Even a map should work just fine.

Namespacing makes sense.

Reply:
Made the change, works nicely I think.
Lemme know if you agree.